README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://dev.azure.com/pagopa-io/io-spid-commons/_apis/build/status/pagopa.io-spid-commons.code-review?branchName=176013578-refactor-pipeline)](https://dev.azure.com/pagopa-io/io-spid-commons/_build/latest?definitionId=99&branchName=176013578-refactor-pipeline)
```

Comment:
I like the idea, but as it's configured it's showing no interesting information. This badge is meant to show if master is shippable (meaning: it builds and tests are ok). No great value for as, as we perform all tests we have everytime we merge.

If we had long-running integration/e2e tests that we run **after** the merge, that would really make more sense.

Reply:
I'll remove it.. actually I need to remove also from io-ts-commons..